docs/source/Userguide.rst

```diff
@@ -0,0 +1,118 @@
+
+Pages in the FIWARE Lab GUI:
+---------------------------------------------
+**Login Page:** User have to login with their credentials to access FIWARE Lab dashboard.
+
+.. figure:: figures/login-page.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+**API Access Page:** User can find Project ID after clicking view credentials in API Access section of FIWARE Lab dashboard.
+
+.. figure:: figures/view-credentials.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+**Instances:** User have to create instances in instances tab of FIWARE Lab dashboard.
+
+.. figure:: figures/instances-page.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+**Overview Page:** User have to click on kubernetes button to redirect to kubernetes GUI for cluser deployment.
+
+.. figure:: figures/overview-page.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+Pages in the kubernetes GUI:
+---------------------------------------------
+
+**Dashboard:** After clicking on button, the default screen will be Dashboard which will list all the clusters created from the user account. User can click on the ID to check and modify the cluster details.
+
+.. figure:: figures/dashboard.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %   
+
+**Create Cluster:** User will get option to create his own cluster in "Create Cluster" tab. User needs to fill the details which will be used for creating cluster.
+
+.. figure:: figures/create-cluster.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+**Components details:** User will get option to View and edit components. For this first user needs to select the Cluster name from the Cluster List and after that user can view/edit its components.
+
+.. figure:: figures/component.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+**Add Node:** User can add node details in "Add Node" option after clicking on the ID shown in the Dashboard. User need to provide details such as VM username (centos/ubuntu), VM IP (internal IP), pem file to access the VM and after submitting the information, VM details will be added to the cluster.
+
+.. figure:: figures/add-node.png
+   :align: center
+   :width: 100px
+   :height: 100px
+   :scale: 50 %
+
+**Add Components:** User can add Fiware GEs to the cluster by using "Add Components" option after clicking on the ID  shown in the Dashboard. User can select the components which he needs in the cluster by selecting the Enable option. User can also select the version of the Fiware GEs and its database version from the dropdown. After submitting the details, Fiware GEs will be added to the cluster.

```

Comment:
Add screenshot with Add Components option

Reply:
Screenshot was added to guide user to add component.